### ISSUE TITLE
the dotted line.

### ISSUE THREAD

**Issue by bangslosan** [dbankier/TiVinci#2]
If there is transparency(strokeAlpha), Drawing lines occur the dotted line.

Help me~~ ^^

**Comment by dbankier** [comment]
Yes this is a know issue. Drawing in a smooth way _as you move_ your finger (in this module) consists of drawing curves between points. With a level of opacity you can see where the curves start and end as they overlap (showing the dots). (You won't get this with straight line drawing.)
